testsuite/features/step_definitions/common_steps.rb

```diff
@@ -299,7 +299,7 @@
   target = $client
   arch, _code = target.run('uname -m')
   arch.chomp!
-  cmd = "zgrep '#{text}' #{client_raw_repodata_dir("test-channel-#{arch}")}/primary.xml.gz"
+  cmd = "zgrep '#{text}' #{client_raw_repodata_dir("test-channel-#{arch}")}/*primary.xml.gz"
```

Comment:
ok, I did not know zgrep. I hope it exists on all clients.

Reply:
we use this since years. So I think the answer is "yes"